content/en/docs/concepts/configuration/secret.md

```diff
@@ -181,45 +181,6 @@ If you want to access data from a Secret in a Pod, one way to do that is to
 have Kubernetes make the value of that Secret be available as a file inside
 the filesystem of one or more of the Pod's containers.
 
-To configure that, you:
```

Comment:
Can you add some comments to these paragraphs articulating what happened to them (moved or deleted)?
For large changes like this one, only the author has a clear idea what has actually happened.